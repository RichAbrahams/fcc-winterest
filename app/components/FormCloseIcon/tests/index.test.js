import React from 'react';
import renderer from 'react-test-renderer';
import FormCloseButton from '../index';

it('renders correctly', () => {
  const tree = renderer.create(
    <FormCloseButton />).toJSON();
  expect(tree).toMatchSnapshot();
});
