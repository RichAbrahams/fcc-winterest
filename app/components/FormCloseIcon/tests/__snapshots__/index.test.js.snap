exports[`test renders correctly 1`] = `
<span
  aria-hidden={true}
  className="fa fa-undefined cRIVAo"
  color={undefined}
  onClick={[Function]}
  transform={undefined} />
`;
